Sources/ApexyLoader/ContentLoader.swift

```diff
@@ -28,8 +29,18 @@ open class ContentLoader<Content>: ObservableLoader {
     public var state: LoadingState<Content> = .initial {
         didSet {
             stateHandlers.forEach { $0.notify() }
+            if #available(macOS 10.15, *), #available(iOS 13.0, *) {
+                statePublisher.send(state)
+            }
         }
     }
+    
+    /// Content loading status. The default value is `.initial`.
+    ///
+    /// - Remark: To change state use `update(_:)`.
+    @available(iOS 13.0, *)
+    @available(macOS 10.15, *)
+    private(set) public lazy var statePublisher = CurrentValueSubject<LoadingState<Content>, Never>(.initial)
```

Comment:
library users can complete the publisher
https://developer.apple.com/documentation/combine/currentvaluesubject/send(completion:)

it seems that this is an unexpected behavior

Reply:
Fixed. Please check it out.